internal/docker/build.go

```diff
@@ -37,6 +37,7 @@ func Build(
 	commonArgs := []string{
 		"buildx",
 		"build",
+		"--load",
```

Comment:
If this doesn't work, let's revert to what was working and not spend more time on this?